amcl/CMakeLists.txt

```diff
@@ -1,141 +1,153 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(amcl)
 
-find_package(catkin REQUIRED
-        COMPONENTS
-            rosbag
-            roscpp
-            tf
-            dynamic_reconfigure
-            nav_msgs
-        )
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
```

Comment:
As discussed in the meeting yesterday we are switching and new packages should switch directly. cf https://github.com/ros2/joystick_drivers/pull/1/files#diff-21982c7922a56bee5085457b5536559aR6